core/io/file_access_pack.h

```diff
@@ -41,6 +41,7 @@
 #define PACK_HEADER_MAGIC 0x43504447
 // The current packed file format version number.
 #define PACK_FORMAT_VERSION 1
+#define ENCRYPTED_PACK_FORMAT_VERSION 666
```

Comment:
Easter egg? ;)
You're not using it in the end, the write code uses `PACK_FORMAT_VERSION`. If there's no reason for them to differ, I'd suggest to keep only that one.

Reply:
Part of the unfinished per-file encryption (with encryption field added to the directory record), for the full-PCK encryption there's reason for it indeed.